suite-common/wallet-core/src/fiat-rates/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const actionPrefix = '@common/wallet-core/fiat-rates';
```

Comment:
please move this const to `fiatRatesActions.ts` and rest of these constants move to thunk file because they are used only there. It's not necessary to decouple thing only because they are constants.